randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -1267,7 +1295,9 @@ Extra - room_id: [45]
       Any of the following:
           Underwater Wall Jump (Expert)
           Gravity Suit and Space Jump
-          Hi-Jump and Can Freeze Enemies
+          All of the following:
+              Can Freeze Enemies
```

Comment:
are we sure we want the freezing to be trickless?

Reply:
yes